react/week-19/lesson.md

```diff
@@ -159,27 +157,28 @@ The last command should open a web browser for you. This shows the application t
 We looked at the beginning of the lesson at the concept of components. Now let's look at how components are made in React.
 
 ```js
-import React from 'react'
-import ReactDOM from 'react-dom'
+import React from 'react';
+import ReactDOM from 'react-dom';
 
 function HelloWorld() {
   return (
     <div>Hello World</div>
   )
 }
 
-ReactDOM.render(<HelloWorld />, document.getElementById('root'))
+ReactDOM.render(<HelloWorld />, document.querySelector('#root'));
 ```
 
-There are 3 important parts in this code:
+There are 4 important parts in this code:
 
-1. First we import `React`. This is important because JSX is converted to `React.createElement` calls. If the `React` variable is undefined then this will fail
-2. We create a React component called `HelloWorld`
-3. We *render* the `HelloWorld` component into a `div` with the id of `root`
+1. First we import `React`. This is important because JSX is converted to `React.createElement` calls. If the `React` variable is undefined then this will fail.
+2. We import `ReactDOM` which will be used to *render* the component on the screen.
```

Comment:
Interestingly `ReactDOM` kept coming up as a problem for class 5 in London.

In all of our interactive examples we have the `ReactDOM.render` right in the same file. Which is different to the student's CRA code - there's already a `ReactDOM.render` implemented in `index.js`.

They thought that it was important to have `ReactDOM.render` in every file. Additionally we use the word "render" a lot to mean returning a component, so it kind of makes sense that you would need a method named `render`.

I found it difficult to explain what `ReactDOM.render` was doing. And how it was different from just regular `React` without going off on a major tangent.

I think I'm leaning towards mostly ignoring it and changing all of the examples to have a split between the code that mounts onto the root node and the "application" code. Basically to mimic the default CRA file structure.